### ISSUE TITLE
Metrics monitoring

### ISSUE THREAD

**Issue by palarax** [kentaroy47/frcnn-from-scratch-with-keras#16]
Is it possible to add Tensorboard to allow for monitoring 

This is what I've done in "train_rpn.py" but i only seem to see Graph

`model_rpn.compile(optimizer=optimizer, loss=[losses.rpn_loss_cls(num_anchors), losses.rpn_loss_regr(num_anchors)],metrics=['accuracy'])`
...
`Tensorboard = TensorBoard(log_dir='logs/{}'.format(time()))
callback=[Callbacks, Tensorboard]
history = model_rpn.fit_generator(data_gen_train,epochs=options.num_epochs, validation_data=data_gen_val,steps_per_epoch=1000,callbacks=callback, validation_steps=10)`

I can see rpn loss history is saved, but how can we graph it

`import numpy
numpy_loss_history = numpy.array(loss_history)
numpy.savetxt(options.network+"_rpn_loss_history.txt", numpy_loss_history, delimiter=",")`

**Comment by kentaroy47** [comment]
ok, let me go through this.

**Comment by mariaculman18** [comment]
Hi @kentaroy47, thank you for making available this project.

This ongoing on metrics monitoring means the mAP estimation would be included in the tensorboard? Is there a present way to measure mAP? Is there a present way to measure loss with a validation dataset?
